src/SFA.DAS.EmployerFinance.MessageHandlers/Program.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Net;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Host;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using NLog.Extensions.Logging;
+using SFA.DAS.AutoConfiguration;
+using SFA.DAS.EmployerFinance.MessageHandlers.DependencyResolution;
+using SFA.DAS.EmployerFinance.Startup;
+
+namespace SFA.DAS.EmployerFinance.MessageHandlers
+{
+    public static class Program
+    {
+        public static async Task Main()
+        {
+            ServicePointManager.DefaultConnectionLimit = 50;
+
+            using (var container = IoC.Initialize())
+            {
+                var startup = container.GetInstance<IStartup>();
+                var environmentService = container.GetInstance<IEnvironmentService>();
+                var environmentName = environmentService.IsCurrent(DasEnv.LOCAL) ? EnvironmentName.Development : EnvironmentName.Production;
+                var jobActivator = new StructureMapJobActivator(container);
+            
+                var host = new HostBuilder()
+                    .UseEnvironment(environmentName)
+                    .ConfigureWebJobs(b => b.AddAzureStorageCoreServices().AddTimers())
+                    .ConfigureAppConfiguration(b => b.AddJsonFile(environmentName).AddEnvironmentVariables())
+                    .ConfigureLogging(b => b.AddNLog())
+                    .ConfigureServices(c => c.AddSingleton<IJobActivator>(jobActivator))
+                    .UseConsoleLifetime()
+                    .Build();
+                
+                var loggerProvider = host.Services.GetService<ILoggerProvider>();
+                container.Configure(c => c.For<ILogger>().Use(x => loggerProvider.CreateLogger(x.ParentType.FullName)));
```

Comment:
This line shouldn't be needed. If we need to inject an `ILogger` in future though it should probably be resolved from an `ILoggerFactory` with registration via a `SFA.DAS.EmployerFinance.DependencyResolution.LoggerRegistry` file e.g.:

```c#
For<ILoggerFactory>().Use(() => new LoggerFactory().AddNLog()).Singleton();
For<ILogger>().Use(c => c.GetInstance<ILoggerFactory>().CreateLogger(c.ParentType));
```

Reply:
Yes, I originally did this to reuse the one created in the host as wasn't sure if having two separate ways to create the NLogger was what we wanted so went with the option that had the smaller footprint and then assumed it would be changed if required. Are you saying you want to remove the logging then rather than keep this in and change when required?

Reply:
Yeah remove it for now and we'll add the registration if/when we need it.

Reply:
ok